app/_features/room/components/conference-lobby.tsx

```diff
@@ -290,32 +305,45 @@ export default function ConferenceLobby({ roomID }: LobbyProps) {
                   </Button>
                 </div>
               </div>
-              <div className="mt-6 h-16">
-                <div className="fixed bottom-0 left-0 w-full border-t border-zinc-800 bg-zinc-900 py-2.5 lg:relative lg:border-t-0">
+            </div>
+
+            {/* Action Buttons */}
+            <div>
+              <div className="fixed bottom-0 left-0 flex w-full flex-col gap-2 border-t border-zinc-800 bg-zinc-900 py-2.5 lg:relative lg:border-t-0">
+                {/* Preparing Room Message */}
+                {isError && (
                   <div className="mx-auto w-full max-w-xl px-4 lg:px-0">
-                    <Button
-                      className="w-full rounded-lg bg-red-700 px-4 py-2 font-semibold text-zinc-200 antialiased hover:bg-red-600 active:bg-red-500"
-                      onClick={openConferenceRoom}
-                      isDisabled={isSubmitting || isError}
-                      aria-disabled={isSubmitting || isError}
-                      disabled={isSubmitting || isError}
-                    >
-                      {isSubmitting ? (
-                        <div className="flex gap-2">
-                          <Spinner
-                            classNames={{
-                              circle1: 'border-b-zinc-200',
-                              circle2: 'border-b-zinc-200',
-                              wrapper: 'w-4 h-4',
-                            }}
-                          />
-                          <span>Joining to room...</span>
-                        </div>
-                      ) : (
-                        <span>Join to Room</span>
-                      )}
-                    </Button>
+                    <div className="flex gap-4 rounded-lg bg-zinc-950 p-4">
+                      <Spinner size={'md'} />
+                      <p className="text-sm">
+                        {`preparing room, please wait. this page is auto-refresh when ready.`}
```

Comment:
@GagahGK  I think we can make this copywriting a bit more professional. How does this sound to you? "**Please wait while we prepare this room. This page will auto-refresh when the room is ready.**"

Is this too long for the UI?

Reply:
I've discussed  this with Ashar, the copy have been changed several times, and it was the final copy from him

Reply:
@GagahGK  At the very least, can you capitalize the letter? "Preparing", and "This page" should be better to see grammatically.

Reply:
Capitalized now : [a070143](https://github.com/inlivedev/inlive-room/pull/148/commits/a070143ce79c47c6c792dfd1bada3ddc050812bb)